pkg-manager/plugin-commands-installation/test/add.ts

```diff
@@ -79,12 +84,16 @@ test('installing with "workspace:" should work even if link-workspace-packages i
     },
   ])
 
+  const workspaceManifest = { packages: ['project-1', 'project-2'] }
+  await writeYamlFile('pnpm-workspace.yaml', workspaceManifest)
+
   await add.handler({
     ...DEFAULT_OPTIONS,
     dir: path.resolve('project-1'),
     linkWorkspacePackages: false,
     saveWorkspaceProtocol: 'rolling',
     workspaceDir: process.cwd(),
+    workspaceManifest: { dir: process.cwd(), contents: workspaceManifest },
```

Comment:
It's a bit annoying that tests calling the `install`/`add`/`import` handlers need to pass in `workspaceManifest` directly.

One way to fix this would be to better distinguish between a `workspaceManifest` field that's (a) not yet read or (b) read, but empty. We would then call `readWorkspaceManifest` in the former case.

But that's a lot of complexity to make a few tests easier to write. I decided not to make the `workspaceManifest` config field more complicated.